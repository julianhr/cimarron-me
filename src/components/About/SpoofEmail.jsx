import React from 'react'
import PropTypes from 'prop-types'
import styled from '@emotion/styled'

import SvgImage from '~/assets/img/suspicious_emoji.svg'


const Root = styled.div`
  align-self: center;
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 80px 0 0;
  text-align: center;

  h2 {
    color: ${props => props.theme.colors.text};
  }
`


function EmailSent() {
  return (
    <Root>
      <h2
        css={{ marginBottom: 80 }}
      >
        That was an awefully suspicious email
      </h2>
      <SvgImage
        width={240}
        height={240}
      />
      <h2
        css={{ marginTop: 80 }}
      >
        I have reported it to my detective minions
      </h2>
    </Root>
  )
}

export default EmailSent
