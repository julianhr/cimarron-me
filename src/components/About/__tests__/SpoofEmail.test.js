import React from 'react'
import { render, mount, shallow } from 'enzyme'
import { create } from 'react-test-renderer'

import SpoofEmail from '../SpoofEmail'
import MockApp from '~/__tests__/__mocks__/MockApp'


describe('SpoofEmail', () => {
  it('matches snapshot', () => {
    const wrapper = render(
      <MockApp>
        <SpoofEmail />
      </MockApp>
    )

    expect(wrapper).toMatchSnapshot()
  })
})
