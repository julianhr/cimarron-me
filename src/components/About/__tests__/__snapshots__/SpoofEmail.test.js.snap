// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpoofEmail matches snapshot 1`] = `
<div
  class="css-1858cwa-Root e1bi310a0"
>
  <h2
    class="css-1latw0z-EmailSent"
  >
    That was an awefully suspicious email
  </h2>
  <test-file-stub
    height="240"
    width="240"
  />
  <h2
    class="css-1rh688y-EmailSent"
  >
    I have reported it to my detective minions
  </h2>
</div>
`;
